build-advisories/test/RoaveTest/SecurityAdvisories/VersionConstraintTest.php

```diff
@@ -277,4 +504,49 @@ function (array $entry) {
             $entries
         );
     }
+
+    /**
+     * @param VersionConstraint $versionConstraint
+     * @param VersionConstraint $other
+     *
+     * @return bool
+     */
+    private function callContains(VersionConstraint $versionConstraint, VersionConstraint $other)
```

Comment:
Agreed, though the methods are pure and achieving full path coverage was a mess here otherwise.

I really wanted to test specific private logic to verify safety of my assumptions, but may move them to public/static operations later on.